cmd/ssl-expiry-check/README.md

```diff
@@ -22,15 +24,17 @@ spec:
       - env:
           # Domain name env variable must be updated to the domain on which you wish to check the SSL for
           - name: DOMAIN_NAME
-            value: "corporate.comcast.net"
+            value: "corporate.comcast.com"
```

Comment:
Maybe sanitize this? (possibly sensitive information)

Reply:
This is our public facing, corporate site. I chose that as opposed to "google.com" because Google seems to renew their certs very frequently and keep them constantly on the verge of expiring, and the default check length of 60 days fails..